docs/logs/faq.md

```diff
@@ -58,12 +58,9 @@ Click the `App Actions` button and select `App Settings`. App Info page will ope
 
 **Can I send old logs?**
 
-Yes, you can send all logs.  However, keep in mind your logging app's
+Yes, you can send all logs. However, keep in mind your logging app's
 Data Retention settings.  For example, if your retention is set to 7
-days and you send logs that are 10 days old (i.e. they contain a
-timestamp from 10 days ago), they will get accepted, but will get
-removed shortly after. Thus, check your app's data retention before
-importing old data.
+days and you send logs that are 10 days old we will give you warning that you are outside of valid retention time range.
```

Comment:
days and you send logs that are 10 days old we will give you warning that you are outside of valid retention time range.
=>
days and you send logs that are 10 days old, we will give you a warning saying that you are outside of your valid retention time range.